main/FloatingActionButtonWrapper/index.tsx

```diff
@@ -0,0 +1,120 @@
+import {DoobooTheme, withTheme} from '../theme';
+import {Icon, IconName} from '../Icon';
+import React, {FC, useMemo} from 'react';
+
+import {Animated} from 'react-native';
+import {IconButton} from '../IconButton';
+import styled from '@emotion/native';
+
+const AbsolutePositionWrapperView = styled.View`
+  position: absolute;
+`;
+
+export const StyledIcon = styled(Icon)`
+  color: ${({theme}) => theme.textContrast};
+`;
+
+const FABItemWrapperView = styled.View`
+  position: relative;
+  margin: 10px;
+`;
+
+export type FABItem = {
+  icon: IconName;
+  onPress: () => void;
+};
+
+interface FloatingActionButtonsWrapperProps {
+  active: boolean;
+  DefaultFAB: FABItem;
```

Comment:
I think this better naming would be `MainFab`. Default sounds like something before customized.